.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://npm.pkg.github.com/pulsarplatform
```

Comment:
I am not sure this commit was intentional

Reply:
Sorry, my bad! It wasn't intentional... i removed that!